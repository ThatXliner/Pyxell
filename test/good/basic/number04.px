
print -0b_1_1110_0011_0101
print -0o_0123456_7
print -0x_BA_93_7E_d4_Ff_52c8e
