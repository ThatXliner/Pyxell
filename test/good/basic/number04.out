-7733
-342391
-840264297869552782
